lldb/test/API/commands/expression/expr_inside_lambda/TestExprInsideLambdas.py

```diff
@@ -54,7 +54,7 @@ def test_expr_inside_lambda(self):
         self.expect_expr("base_var", result_type="int", result_value="14")
         self.expect_expr("this->shadowed", result_type="int", result_value="-1")
         
-        # 'p this' should yield 'struct Foo*'
+        # 'expression this' should yield 'struct Foo*'
```

Comment:
Did you make these changes upstream, too?

Reply:
You may need to resolve fewer merge conflicts if you cherry-pick the change first

Reply:
I backed this change out. Will do upstream.